### ISSUE TITLE
Training new models outputs unacceptable results

### ISSUE THREAD

**Issue by valentinvieriu** [genekogan/CubistMirror#1]
I'm trying to generate a new model for more than a week, and I'm getting weird unacceptable results.
See for yourself:
![image](https://cloud.githubusercontent.com/assets/478866/16374042/045cd2d0-3c55-11e6-9d2b-62f83c9610d7.png)
or like:
![image](https://cloud.githubusercontent.com/assets/478866/16374060/13b8f772-3c55-11e6-80a9-6f2e4e52a088.png)

I see that you also get this border around the generated image, but somehow you mange to pull a decent looking solution. Can you share if you changed anything in the code to get your cubist.model?

Thank  you

**Comment by genekogan** [comment]
yep this is roughly what i was getting. it should be expected that the quality is much less than the original style transfer unfortunately because it uses a fast method instead of the optimization based approach favored by the original paper.

i didn't change the training code at all, although i trained it for up to 10 iterations instead of just 2 as the repo suggested. it looks mostly the same, but i found the extra training time helped to reduce the weird artifacts (not the ones around the border though, those i suspect are due to windowing size issues).  another thing i found nice is to reduce the size of the input image -- this makes it faster and to my eyes it looks better, at least for the cubist model.

a lot of styles end up not looking very good. cubism works well because of the obvious features (hard edges and corners). i tried hokusai also but it was junk. not much we can do about it at the moment.

**Comment by valentinvieriu** [comment]
Thank you for the input @genekogan. I will try couple of different input images. Can you share your cubism image with us? I'm curious about the size. Also did you resized the train data also? In the paper it is mentioned that was resized to 320px. I've tried that one too without any improvements.

**Comment by valentinvieriu** [comment]
Hi @genekogan ,
I've posted a request on https://github.com/yusuketomoto/chainer-fast-neuralstyle/issues/1#issuecomment-228944706 and @yusuketomoto mentioned that I should try to modify the batchsize to 1 and then the output should look much better.
I've given that a try and must say it does look better. The border is still there unfortunately.
I would like to rebuild the cubist model and train it with the new batchsize.
Can you add the original cubist image to the repo so I can create a pull request and send it back to you. 

Also I'm thinking on contributing back with other models that I'm training. Let me know if you consider this valuable.

**Comment by genekogan** [comment]
interesting, give it a shot. here's the original:

![cubist](https://cloud.githubusercontent.com/assets/1335251/16408118/70b9be6c-3d17-11e6-976b-d35cc10b08c8.jpg)

yes, definitely models would be great. we can host them somewhere and i'd link to it from the repo, or you can PR it directly to the repo if you wish.

curious to see how it turns out.

**Comment by gafr** [comment]
@genekogan I [started a repository](https://github.com/gafr/chainer-fast-neuralstyle-models) where I want to collect models. Sneaky as I am, I included your models directly in the repository and put a reference to your page. If you want I remove the models and just point to your repo.

**Comment by genekogan** [comment]
@gafr this is great! i've put a reference in my doc. thanks for collecting these. are you planning to retrain them for the updated repo?
